documentation/source/user_section/installation/windows.rst

```diff
@@ -343,53 +327,32 @@ First, save your Docker image:
-  - Try again
+4. Install `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_.
 
-To check that X forwarding is working well write ``fsleyes &`` in LXterminal and FSLeyes should open, depending on how fast your computer is FSLeyes may take a few seconds to open. If fsleyes is not working in the LXterminal:
+5. Launch an X11 Server with XLaunch
 
-- Check if it's working on the docker machine by running ``fsleyes &`` in the docker quickstart terminal
-- If it works, run all the commands in the docker terminal.
-- If it throws the error ``Unable to access the X Display, is $DISPLAY set properly?`` follow these next steps:
+- Run XLaunch, which should have been installed by default.
+- Check 'Multiple Windows' and set the **display number** to ``0``, which you will need later. (The default display number ``-1`` will automatically detect the display number, unless you are running a setup with multiple monitors it will typically use ``0``)
+- Then, you can click Next, select 'Start no Client' then click Next
+- **Uncheck** 'Native opengl' and **check** 'Disable Access Control' then click Next, then click Finish.
 
-  - Run ``echo $DISPLAY`` in the LXterminal
-  - Copy the output address
-  - Run ``export DISPLAY=<previously obtained address>`` in the docker quickstart terminal
-  - Run ``fsleyes &`` (in the docker quickstart terminal) to check if it is working. A new Xming window with fsleyes should appear.
+6. Determine the IPv4 address of the virtual Ethernet Adapter by running 'ipconfig' in Powershell or the Command Prompt, then looking at the ``Ethernet adapter vEthernet (WSL)`` entry.
 
-Notes:
+7. In your Terminal window, run the following command, filling in the IP address, display number, and image name noted earlier:
+   
+   .. code:: bash 
+   
+      docker run -it --rm -e DISPLAY=<IPv4_ADDRESS>:<DISPLAY_NUMBER> -e XDG_RUNTIME_DIR=/tmp/runtime-root <IMAGE_NAME>/ubuntu:ubuntu22.04
 
-- If after closing a program with graphical interface (i.e. FSLeyes) LXterminal does not raise the shell ($) prompt then press Ctrl + C to finish closing the program.
-- Docker exposes the forwarded SSH server at different endpoints depending on whether Docker Desktop or Docker Toolbox is installed.
 
-  - Docker Desktop:
-
-    .. code:: bash
-
-       ssh -Y -p 2222 sct@127.0.0.1
-
-  - Docker Toolbox:
-
-    .. code:: bash
-
-       ssh -Y -p 2222 sct@192.168.99.100
+8. You can test whether GUI scripts are available by running the following command in your Docker container:
+ 
+   .. code:: bash
+   
+      sct_check_dependencies
+      
+   You should see two green ``[OK]`` symbols at the bottom of the report for "PyQT" and "matplotlib" checks, which represent the GUI features provided by SCT. 
+   
+If you plan to use a web server within the container to access SCT output from the host's browser, be sure to map your ports by including the ``-p <Host Port>:<Container Port>`` flag in the above command.
```

Comment:
> There I explained how to map ports between the host machine and the docker container, perhaps for the purposes of serving `.html` output from the container so that the host machine can access it the output through its web browser.
> 
> Is this a valid use case for SCT with Docker or would it be better to omit these instructions?

SCT has a "QC Report" feature that uses the web browser. However, it doesn't run a web server -- instead, produces an `index.html` file that viewers can open in their browser. So, I think it is more likely that the user would just try to copy the QC folder (with `index.html`) from the container to the local machine using `docker cp`, then open the `index.html` file normally.

I think that you probably can go ahead and omit the instructions. Thank you for double-checking! :hearts:

```suggestion
```